docs/reviewing.md

```diff
@@ -105,15 +105,20 @@ def _configure_cmake(self):
 
 ### Minimalistic Source Code
 
-The contents of `test_package.c` or `test_package.cpp` should be as minimal as possible, including a few headers at most with simple instatiation of objects to ensure linkage
+The contents of `test_package.c` or `test_package.cpp` should be as minimal as possible, including a few headers at most with simple instantiation of objects to ensure linkage
 and dependencies are correct.
 
 ### Verifying Components
 
 When components are defined in the `package_info` in `conanfile.py` the following conditions are desired
 
-- use the `cmake_find_package` or `cmake_find_package_multi` generators in `test_package/conanfile.py`
-- corresponding call to `find_package()` with the components _explicitly_ used in `target_link_libraries`
+* use `cmake_find_package` if library has an [official](cmake.org/cmake/help/latest/manual/cmake-modules.7.html#find-modules) CMake module emulated in the recipe.
```

Comment:
What if a project installs a cmake script without targets or the documentation of CMake doesn't mention a target?
Is it okay to use cmake variables then? Eventually emulated with a included custom cmake module?

Reply:
does it? there is no word `target` used, only word `module` (perhaps, meaning `find module`, as hyperlink suggests), which is way more generic, and includes both modules with variables and modules with targets.

Reply:
The next sentence explicitly called out targets... just not in the UI diff

Reply:
the next sentence is about `cmake_find_package_multi`, which is all about targets